### ISSUE TITLE
crash report

### ISSUE THREAD

**Issue by alexfromgeneva** [avocode/avocode#1709]
* **Version**: 2.16.0
* **Sketch plugin version**: 47.1.0

* **Web/Desktop**: Desktop

* **Platform**: darwin

* **Avocode UID**: 194329
---

```
Error: Could not find upload info for artboard AC13CA81-ABEE-4545-B5D1-9AE375C39D72
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3352020
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.forEach (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4383:19)
    at t._handleUserDataDesignsRequest (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3351886)
    at emitTwo (events.js:106:13)
    at t.emit (events.js:194:7)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1134058)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1491381
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.forEach (/Applications/
```

**Comment by connor-bean** [comment]
Hi, thanks for reporting this. We're already tracking this problem in #1679, it should be fixed in 2.16.2 this week.

**Comment by alexfromgeneva** [comment]
Hi,

thanks for letting me know!

a




[image: --]

Alex Faundez
[image: http://]about.me/alex.faundez
<http://about.me/alex.faundez?promo=email_sig>

Content strategist and online communications specialist
faundez.alex@gmail.com

Founder of Geneva Content Strategy Meetup
<http://www.meetup.com/Content-Strategy-Geneva/?utm_source=GmailSignature&utm_medium=email&utm_campaign=signature>
